src/objects/GS_Admin_Checklist_Item__mdt.object

```diff
@@ -153,22 +157,22 @@
     </fields>
     <fields>
         <fullName>Secondary_Button_Label__c</fullName>
-        <description>The api name for the Secondary Button in a section item on the Get Started page.</description>
+        <description>The API name for Secondary Button label in a section item on the Get Started page. Must match an imported label in the gsLabelMapper LWC.</description>
         <externalId>false</externalId>
         <fieldManageability>DeveloperControlled</fieldManageability>
-        <inlineHelpText>The api name for the Secondary Button in a section item on the Get Started page.</inlineHelpText>
-        <label>Secondary Button Label</label>
+        <inlineHelpText>The API name for Secondary Button label in a section item on the Get Started page. Must match an imported label in the gsLabelMapper LWC.</inlineHelpText>
+        <label>Secondary Button Label Name</label>
         <length>255</length>
         <required>false</required>
         <type>Text</type>
         <unique>false</unique>
     </fields>
     <fields>
         <fullName>Secondary_Button_Type__c</fullName>
-        <description>The type of action related to the Secondary Button. Indicates if the button is disabled, opens a new Salesforce tab, or opens an external page.</description>
+        <description>The API name for Secondary Button label in a section item on the Get Started page. Must match an imported label in the gsLabelMapper LWC.</description>
```

Comment:
Shouldn't this be "The type of action related to the Secondary Button. Indicates if the button is disabled, opens a new Salesforce tab, or opens an external page."